concepts/server.ts

```diff
@@ -1,11 +1,13 @@
 import createApp from "./src/app";
 import { getElasticClient } from "./src/services/elasticsearch";
 
+const pipelineDate = "2022-05-30";
+
 getElasticClient({
-  serviceName: "concepts_api",
-  pipelineDate: "2022-05-30",
+  serviceName: "catalogue_api", // TODO an elasticsearch user for the concepts API
```

Comment:
I assume this is blocked on provisioning ES users in TF or similar?

Reply:
Indeed